pkg/hub/controllers/cluster/reconciler.go

```diff
@@ -272,6 +270,15 @@ func (r *Reconciler) getCluster(ctx context.Context, req reconcile.Request) (*hu
 	return hubCluster, nil
 }
 
+func isEmptyString(nodeIPs []string) bool {
```

Comment:
The method name is confusing. How about we invert the logic and call it `isValidNodeIpList` or `isNonEmptyArray`?